presto-docs/src/main/sphinx/release/release-0.273.rst

```diff
@@ -0,0 +1,80 @@
+* Add a Presto connector for Clickhouse with support of username/password authentication.
+* Improve performance of :func: `zip_with` when used inside a :func:`reduce_agg`.
+* Change protoc and grpc version in presto-grpc-api module to adapt to linux aarch64.
+* Upgrade zstandard compression to version 1.5.2.2. This improves the cpu used for data compressed with zstandard by about 2%.
+* Enable retrying max requests queued exception.
+* Enable warnings for approx_distinct and approx_set with low ``MAX_STANDARD_ERROR``.
+* Enhance assertQuery to support actual and expected session parameters
+* Throw ``NOT_SUPPORTED`` PrestoException if the SQL statement has an offset clause without ``offset_clause_enabled`` being set
+* Allow connectors to support column subfield access control checks. Connectors can specify subfield access control for row-type columns through ``checkCanSelectFromColumns()`` in SPI.
+* Upgrade postgresql driver to 42.3.3.
+
+Verifier Changes
+________________
+* Add support to verifier for running control and test queries concurrently through config ``concurrent-control-and-test`` and test property.
+
+SPI Changes
+___________
+* Add ``getInfoMap`` method in ``ConnectorSplit`` which returns a ``Map<String, String>``. This method should be preferred to the ``getInfo`` method which returns a raw object.
+
+Hive Changes
+____________
+* Add a limit that prevents caching partitions with column counts greater than a configured threshold. This threshold can be set using the ``hive.partition-cache-column-count-limit`` configuration parameter.
+* Add metastore config property ``hive.metastore.thrift.delete-files-on-table-drop`` to delete files on drop table.
+* Support for overwriting existing partitions with a hive config property ``hive.insert-existing-partitions-behavior``. The config supersedes a legacy config ``hive.insert-overwrite-immutable-partitions-enabled``. The new config adds capability of overwriting new partitions for S3
+* Replace Hive session property ``streaming_aggregation_enabled`` with  ``order_based_execution_enabled``.
+* Replace Hive config ``hive.streaming-aggregation-enabled`` with ``hive.order-based-execution-enabled``.
+* If Hive files are changed, the modification time can be used to invalidate parquet metadata cache immediately.
+
+Iceberg Changes
+_______________
+* Add ``$properties`` system table.
+* Store column comments for iceberg tables.
+* Upgrade Iceberg to 0.13.1.
+* Fix Iceberg $files table in case of column dropping.
+
+Mongodb Changes
+_______________
+* Add :func:`CAST(ObjectId() as STRING)`.
+
+Pinot Changes
+_____________
+* Add double-quotes to Pinot generated queries to ensure that reserved keywords are escaped.
+* Add TLS support in Pinot gRPC connection.
+* Turn on config ``pinot.pushdown-topn-broker-queries`` by default to push down aggregation group by order by queries to improve query performance.
```

Comment:
Improve query performance by enabling pushdown of topn broker queries by default.